target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.h

```diff
@@ -79,12 +94,25 @@
 #define RTL_MC_DACCR				(0x1500)
 #define RTL_MC_DCDR				(0x1060)
 
+#define RTL930X_SYS_STATUS			(0x0044)
+#define RTL930X_PLL_MEM_CTRL0			(0x0234)
+#define RTL930X_PLL_MEM_CTRL1			(0x0238)
+#define RTL930X_PLL_MEM_CTRL2			(0x023c)
+#define RTL930X_PLL_MEM_CTRL3			(0x0240)
+#define RTL930X_PLL_MEM_CTRL4			(0x0244)
+#define RTL930X_PLL_MEM_CTRL5			(0x0248)
+#define RTL930X_PLL_MEM_CTRL6			(0x024c)
+
 #define RTL_MC_MCR_DRAMTYPE(v)			((((v) >> 28) & 0xf) + 1)
 #define RTL_MC_DCR_BUSWIDTH(v)			(8 << (((v) >> 24) & 0xf))
 
+#define RTL930X_PLL_MEM_CTRL2_PREDIV(v)		(((v) >> 14) & 0x3)
+#define RTL930X_PLL_MEM_CTRL3_CMU_NCODE_IN(v)	(((v) >> 24) & 0xff)
+
 /*
- * Other stuff
+ * SRAM stuff
  */
 
 #define RTL_SRAM_MARKER				(0x5eaf00d5)
 #define RTL_SRAM_BASE_CACHED			(0x9f000000)
+#define RTL_SRAM_BASE_UNCACHED			(0xbf000000)
```

Comment:
You've removed the `sync` ops earlier, shouldn't you use uncached memory there as well then?

Reply:
sync is a no longer needed leftover from c compiled sources that I copy/pasted to assembler.

Reply:
Why are they no longer needed though? I don't recall reading that in the commit message.